snapshots/runner.js

```diff
@@ -24,6 +24,7 @@ function runTests(r) {
         const options = {
           lang: tc.lang,
           style: tc.style,
+          scale: 1,
```

Comment:
We didn’t accidentally make `scale` required, did we?

Reply:
It was causing test failure because the rendered image was 0x0. It seems like for `renderMatching` it has a fallback of 1, but for raw APIs it seems like the default is 0.

Reply:
Oops, we should open a new issue to track that 🙈

Reply:
#384